app/utils/validation_utils.py

```diff
@@ -1,9 +1,14 @@
 import re
 
+name_regex = r"(^[A-Za-z\s]+$)"
```

Comment:
Does this allow spaces between words? Also, I think some names can contain dashes "-", can you include that in the regex? And then add a test to this?

Reply:
Yeah It allows spaces between words. I can include "-" in the regex but I doubt if we should even do that because first, normally names doesn't contains "-", second, if user registers with just "-" he/she will get registered and would make the data inconsistent.

Reply:
@GeVic we could also use that argument for a user putting data like "aaaaaaaa", right? I mentioned you in a thread where I started a conversation about dashes on names. one of the community members confirmed that in their country its a common thing to use dash in full name and other commnutity member gave a little more thought on this, you can check it out. What do you think?